### PR TITLE
API: HTMLIFrameElement migration from MDN page and subpages

### PR DESCRIPTION
**Pull request by SphinxKnight**
i-dont-always-make-a-pr-but-when-i-do-the-file-is-2455-lines-long

### CODE REVIEW

File: api/HTMLIFrameElement.json [hunk 1]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "align": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `align` is part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/align

File: api/HTMLIFrameElement.json [hunk 2]
```diff
@@ -0,0 +1,2454 @@
...
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "allow": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/allow",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `allow` does not have its own page.

Suggest you replace the `mdn_url` with a `description`:

```
"allow": {
      "__compat": {
        "description": "<code>allow</code> property",
```

File: api/HTMLIFrameElement.json [hunk 3]
```diff
@@ -0,0 +1,2454 @@
...
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: According to `https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement` the `allow` property is supported by Chrome 60.

File: api/HTMLIFrameElement.json [hunk 4]
```diff
@@ -0,0 +1,2454 @@
...
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: Should be `"version_added": "47"`

File: api/HTMLIFrameElement.json [hunk 5]
```diff
@@ -0,0 +1,2454 @@
...
+          "deprecated": true
+        }
+      }
+    },
+    "allow": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/allow",
+        "support": {
+          "webview_android": {
+            "version_added": null
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `"version_added": "60"`

File: api/HTMLIFrameElement.json [hunk 6]
```diff
@@ -0,0 +1,2454 @@
...
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `"version_added": "60"`

File: api/HTMLIFrameElement.json [hunk 7]
```diff
@@ -0,0 +1,2454 @@
...
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: Should be `"version_added": "47"`

File: api/HTMLIFrameElement.json [hunk 8]
```diff
@@ -0,0 +1,2454 @@
...
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I'm not finding a reference to the `allow` property being experimental.

File: api/HTMLIFrameElement.json [hunk 9]
```diff
@@ -0,0 +1,2454 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "allowFullScreen": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `allowFullScreen` does not have its own page.

Suggest you replace the `mdn_url` with a `description`:


```
"allowFullScreen": {
      "__compat": {
        "description": "<code>allowFullScreen</code>",
```

File: api/HTMLIFrameElement.json [hunk 10]
```diff
@@ -0,0 +1,2454 @@
...
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/allowFullScreen",
+        "support": {
+          "webview_android": {
+            "prefix": "-webkit",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `prefix` should be `webkit` here. (Without the dash).

File: api/HTMLIFrameElement.json [hunk 11]
```diff
@@ -0,0 +1,2454 @@
...
+            "version_added": "17",
+            "notes": "Daily test builds only."
+          },
+          "chrome": {
+            "prefix": "-webkit",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `prefix` should be `webkit` here. (Without the dash).

File: api/HTMLIFrameElement.json [hunk 12]
```diff
@@ -0,0 +1,2454 @@
...
+            "version_added": "17",
+            "notes": "Daily test builds only."
+          },
+          "chrome_android": {
+            "prefix": "-webkit",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `prefix` should be `webkit` here. (Without the dash).

File: api/HTMLIFrameElement.json [hunk 13]
```diff
@@ -0,0 +1,2454 @@
...
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "18"
+            },
+            {
+              "prefix": "-moz",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `prefix` should be `moz` here. (Without the dash).

File: api/HTMLIFrameElement.json [hunk 14]
```diff
@@ -0,0 +1,2454 @@
...
+              "version_added": "9"
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "18"
+            },
+            {
+              "prefix": "-moz",
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `prefix` should be `moz` here. (Without the dash).

File: api/HTMLIFrameElement.json [hunk 15]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "frameBorder": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I'm not finding `frameBorder` anywhere in the `HTMLIFrameElement` documentation.

File: api/HTMLIFrameElement.json [hunk 16]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "longDesc": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `longDesc` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/longDesc
I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 17]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "marginHeight": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `marginHeight` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/marginHeight

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 18]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "marginWidth": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `marginWidth` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/marginWidth

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 19]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "name": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `name` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/name

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 20]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "sandbox": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `sandbox` is any longer part of the HTMLIFrameElement API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/sandbox

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 21]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "scrolling": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `scrolling` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/scrolling

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 22]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "src": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `src` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/src

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 23]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "srcdoc": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: `srcdoc` does not appear to have its own page.

Suggest you replace the `mdn_url` with a `description`:


```
"srcdoc": {
      "__compat": {
        "description": "<code>srcdoc</code>",
```

File: api/HTMLIFrameElement.json [hunk 24]
```diff
@@ -0,0 +1,2454 @@
...
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "width": {
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: I don't think `width` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/width

I'm not finding any data on it.

File: api/HTMLIFrameElement.json [hunk 25]
```diff
@@ -0,0 +1,2454 @@
...
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 26]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 27]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 28]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 29]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 30]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 31]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 32]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 33]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 34]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 35]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 36]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 37]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 38]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 39]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 40]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 41]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 42]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 43]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 44]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 45]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 46]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 47]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 48]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 49]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 50]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 51]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 52]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 53]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

File: api/HTMLIFrameElement.json [hunk 54]
```diff
@@ -0,0 +1,2454 @@
...
```
Comment (maboa) [inline api/HTMLIFrameElement.json]: should be: `"standard_track": false,`

**Comment by jpmedley** [comment]
@SphinxKnight Did you use [Mark Dittmer's tool](https://github.com/mdittmer/mdn-confluence) to generate this? There's much more data to be had here. If you want to kill this PR, I can generate one with more info.

**Comment by SphinxKnight** [comment]
@maboa @jpmedley thanks a lot for reviewing. I'll go through this over this week-end.
@jpmedley I didn't use the tool but I'll do and check if this is worth merging or opening another PR.

Again thanks :)

**Comment by maboa** [comment]
No worries @SphinxKnight - ping me if you have any queries :)

**Comment by SphinxKnight** [comment]
@maboa I've gone through your comments :) 
@jpmedley I've tried, I promise but https://github.com/mdittmer/mdn-confluence/issues/23 Is there something I can do about this one? Do you think it would be better for me to (1) wait for it? (2) go as it is? (3) ask you for a `mdn-confluence` generated file so that I can merge locally?

**Comment by jpmedley** [comment]
This PR is getting old and I worry about old PRs. We can merge and I'll submit a separate PR after it gets in master.

Thanks for trying mdn-confluence and submitting a ticket.

**Comment by SphinxKnight** [comment]
@jpmedley @maboa I've taken Joe's comment into account. For property/method that are not in standards anymore I've put `deprecated: true` though I'm not sure if that's enough. Should I add a `standard_track: false`too ?

**Comment by SphinxKnight** [comment]
Closing this one as it seems it has gotten too big to not cause a "GH Unicorn" most of the time.
I'll reopen another one with the same content.

**Pull request closed without merging**